package.json

```diff
@@ -24,18 +24,23 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
+    "kuzzle-common-objects": "^3.0.0",
```

Comment:
You're right. At the minimum it can be moved to dev-dependencies and maybe totally removed.

Putting the PR in wip in the meantime